Users/Install.rst

```diff
@@ -183,6 +183,7 @@ virtualenv with:
 
 Using coala With Docker
 =======================
+Don't know what Docker is? Have a look at `What is Docker? <https://docs.coala.io/en/latest/Users/Docker_Image.html>`_
```

Comment:
Use relative links

Reply:
I've made the changes, have a look now. Thanks for reviewing :+1: